runtime/src/main/scala/akka/grpc/javadsl/package.scala

```diff
@@ -0,0 +1,21 @@
+package akka.grpc
+
+package object javadsl {
+  /**
+   * Helper for creating akka.japi.function.Function instances from Scala
+   * functions as Scala 2.11 does not know about SAMs.
+   */
+  def japiFunction[A, B](f: A => B): akka.japi.function.Function[A, B] =
+    new akka.japi.function.Function[A, B]() {
+      override def apply(a: A): B = f(a)
+    }
+
+  /**
+   * Helper for creating java.util.function.Function instances from Scala
+   * functions as Scala 2.11 does not know about SAMs.
+   */
+  def javaFunction[A, B](f: A => B): java.util.function.Function[A, B] =
+    new java.util.function.Function[A, B]() {
+      override def apply(a: A): B = f(a)
+    }
+}
```

Comment:
These aren't found in any java compat lib? Then :+1:

Reply:
I could not find them anywhere. Have you seen something like this around? :)